package.json

```diff
@@ -12,7 +12,16 @@
     "lint:test-app": "yarn workspace test-app run lint",
     "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:fix:*",
     "lint:fix:addon": "yarn workspace ember-cookies run lint:fix",
-    "lint:fix:test-app": "yarn workspace test-app run lint:fix"
+    "lint:fix:test-app": "yarn workspace test-app run lint:fix",
+    "prepare": "yarn workspace ember-cookies run build",
```

Comment:
this is needed to build the addon as v2 addon needs to be prebuilt before consumption